packages/commons-test/test/jwt.unit.test.ts

```diff
@@ -305,7 +305,7 @@ describe("JWT tests", () => {
 
       expect(() => readAuthDataFromJwtToken(token!)).toThrowError(
         invalidClaim(
-          "Validation error: Invalid discriminator value. Expected 'm2m' | 'm2m-admin' | 'internal' | 'maintenance' |  at \"role\""
+          "Validation error: Invalid discriminator value. Expected 'internal' |  | 'm2m' | 'm2m-admin' | 'maintenance' at \"role\""
```

Comment:
For this test and other tests in this file, I would make them more flexible by using the approach followed by Alessio here: [#1789](https://github.com/pagopa/interop-be-monorepo/pull/1789#discussion_r2099747451)